### PR TITLE
Refactor clsf and hint services windup 1067

### PR DESCRIPTION
**Pull request by OndraZizka**
I recommend to go through the commits one by one.

### CODE REVIEW

File: reporting/api/src/main/java/org/jboss/windup/reporting/service/MapSumEffortAccumulatorFunction.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package org.jboss.windup.reporting.service;
+
+
+import com.tinkerpop.blueprints.Vertex;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ *
```
Comment (jsight) [inline reporting/api/src/main/java/org/jboss/windup/reporting/service/MapSumEffortAccumulatorFunction.java]: Needs javadoc

Comment (OndraZizka) [inline reporting/api/src/main/java/org/jboss/windup/reporting/service/MapSumEffortAccumulatorFunction.java]: Fixed:
 * This implementation holds a Map, and for each vertex, it increases the key
 * that is determined from the vertex by supplied implementation of vertexToKey({@link Vertex}).

**Comment by OndraZizka** [comment]
I forgot to put back the the frame type if

**Comment by jsight** [comment]
Merged: https://github.com/windup/windup/commit/8275a9d44b9d4a84570ed4dcf6d06cde5b75dfb7

**Pull request closed without merging**